### ISSUE TITLE
[REST] Return UUID of created object when UUID was not provided by client in POST method 

### ISSUE THREAD

**Issue by stefan-ho** [SORMAS-Foundation/SORMAS-Project#4891]
### Situation Description
The REST API only provides an OK code as return when a new object is created via the POST method. In the case the client does not provide a UUID for the new object, it would be helpful to return the generated UUID as a reference to the created object.  

### Feature Description
The POST method returns the UUID generated by SORMAS as the push result when no UUID was provided by the client. This would allow clients to make use of the SORMAS UUID generation and only retrieve the generated UUID of the created object as a reference for further processing.

**Comment by JonasCir** [comment]
That's strange b/c I raised the same issue some time ago and it was fixed in this PR https://github.com/hzi-braunschweig/SORMAS-Project/pull/4086 which I tested and reviewed myself :D

**Comment by stefan-ho** [comment]
@JonasCir 
Correct, this feature request is only about **returning** the generated UUID in the response.
